lib/opte/src/d_error.rs

```diff
@@ -46,7 +35,7 @@ pub trait DError {
     fn leaf_data(&self, _data: &mut [u64]) {}
 }
 
-static_cstr!(EMPTY_STRING, b"\0");
+static_cstr!(EMPTY_STRING, c"");
```

Comment:
Is it worth it to have a macro, now that it's the same as a one-liner?
```suggestion
static EMPTY_STRING: &CStr = c"";
```

Reply:
Probably not, particularly since it's only used for test cases.

Reply:
Done!